### PR TITLE
Add the languages that I am using to contact section 

### PR DESCRIPTION
**Pull request by Hiresh23**
## Purpose
To add languages and icon
#89

## Goals
People can know in which languages they can contact

## Approach
Just add a section in which a paragraph tag is used for text and image tag for logo

### Screenshots
Screenshot 2022-10-15 120711

  
### Preview Link

https://pr-106-new-yoshitha-website.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Change the topic of this pr as the topic of the issue

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Put the preview link as this https://pr-106-new-yoshitha-website.surge.sh/

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
For the issue number in the purpose don't put it in <> tags

**Comment by Hiresh23** [comment]
Done

File: dark/assets/css/styles.css [hunk 1]
```diff
@@ -923,3 +931,28 @@ html {
     bottom: -26px;
     left: calc(50% - 20px);
 }
+
+#contact-section .context1{
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:935]: Remove these early commits

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html [hunk 2]
```diff
@@ -231,6 +231,13 @@ <h3>ABOUT <span class="text-info">ME</span></h3>
       <section id="contact-section">
             <h1>CONTACT ME</h1>
             <h3>CONTACT <span class="text-info">ME</span></h3>
+             <section class="context1"><p>Want to get in touch with me? Fill out the form with your inquiry or mail me,you can also connect with me on social media platforms on above given links</p>
```
Comment (YoshithaRathnayake) [inline dark/index.html:234]: Remove these early commits

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by Hiresh23** [comment]
Bro please remove spam

**Comment by Hiresh23** [comment]
I would work on the website my exam were going so i could not do

**Comment by Hiresh23** [comment]
now what is the use i have been disqualified because of you

**Comment by Hiresh23** [comment]
too late i have been disqualified because of you

**Comment by YoshithaRathnayake** [comment]
> too late i have been disqualified because of you

So you should have tell it to me early

**Comment by YoshithaRathnayake** [comment]
> too late i have been disqualified because of you

If you want create a new github account and do hacktobefest no worries.

**Comment by YoshithaRathnayake** [comment]
> too late i have been disqualified because of you

create new account I will help you @Hiresh23

**Comment by YoshithaRathnayake** [comment]
> too late i have been disqualified because of you

don't tell me bcs of you bcs you should have tell it to me early like I am having exams so I can't do this. If you told like that I won't put the spam label @Hiresh23

**Comment by YoshithaRathnayake** [comment]
> > too late i have been disqualified because of you
> 
> don't tell me bcs of you bcs you should have tell it to me early like I am having exams so I can't do this. If you told like that I won't put the spam label @Hiresh23

I don't know no you reason what are you not doing so should tell it to me otherwise I will put spam label

**Comment by YoshithaRathnayake** [comment]
If you disqualified or if you have exams they both are same bcs if its disqualified you can't do hacktoberfest, if you face your exams you won't be able to do hacktoberfest so no need to worry about this how ever you won't be able to do hacktoberfest @Hiresh23

**Pull request closed without merging**